parse/parse_test.go

```diff
@@ -71,6 +71,20 @@ func TestParse(t *testing.T) {
 		t.Fatalf("expected DefaultName to be ReturnsNilError")
 	}
 
+	if info.DeinitFunc == nil {
```

Comment:
Since Deinit is the less common case, probably want to test where it's not universally set?

Reply:
I mean we're the only user of deinit, but by adding to parse_test this mostly tests when deinit is set, not when it isn't set.

Reply:
https://github.com/magefile/mage/pull/345/files#diff-daea2a9086a9f20e21f1ee99180680de0b4f12ce020fd4c99914b466db5a2c9dR130 tests that Deinit is not set by default - is that what you meant?